.gitattributes

```diff
@@ -0,0 +1 @@
+npm-shrinkwrap.json binary
```

Comment:
This will prevent git from attempting to merge `npm-shrinkwrap.json` as the only way to fix this file is to run `make shrinkwrap`.